readme.txt

```diff
@@ -1,3 +1,5 @@
-This repository contains a Meson build definition for project eigen.
```

Comment:
Library is called http://eigen.tuxfamily.org according to upstream if I understand correctly.